CIME/case/case_run.py

```diff
@@ -292,16 +292,12 @@ def _post_run_check(case, lid):
     ###############################################################################
 
     rundir = case.get_value("RUNDIR")
-    model = case.get_value("MODEL")
     driver = case.get_value("COMP_INTERFACE")
-    model = get_model()
 
-    fv3_standalone = False
+    comp_standalone, model = is_comp_standalone(case)
 
-    if "CPL" not in case.get_values("COMP_CLASSES"):
-        fv3_standalone = True
     if driver == "nuopc":
```

Comment:
@rljacob perhaps moving the line 297 to after 299 will solve the problem?

Reply:
I can try that.